docs/content/en/samples/taggers/tagTemplate.yaml

```diff
@@ -0,0 +1,15 @@
+build:
+  tagPolicy:
+    tagTemplate:
+      template: "{{.FOO}}_{{.BAR}}"
```

Comment:
this is still reading really awkwardly to me. i actually think it might be a bit cleaner to just switch `tagTemplate` and `template`:

```yaml
tagPolicy:
  template:
    tagTemplate: ...
```

so then it would be referred to as the "template tagger" and not the "tag template tagger". WDYT?

Reply:
This doesn't really make sense to me unless you wanted to do something like this as well.
```
tagPolicy:
  template:
    envTemplate: ...
```

Reply:
yeah i guess in the context of mirroring the envTemplate it makes more sense. it's mostly just the amount of times i say "tag" in my head when i read out the config haha. i don't mind `customTemplate`, wanna just go with that?

Reply:
Sounds good to me!